server/config/config.go

```diff
@@ -855,12 +856,16 @@ func (c *DRAutoSyncReplicationConfig) adjust(meta *configutil.ConfigMetaData) {
 // MicroServiceConfig is the configuration for micro service.
 type MicroServiceConfig struct {
 	EnableSchedulingFallback bool `toml:"enable-scheduling-fallback" json:"enable-scheduling-fallback,string"`
+	EnableTSOFallback        bool `toml:"enable-tso-fallback" json:"enable-tso-fallback,string"`
```

Comment:
Is this neccesary? The code about this feature is so big that, I think, this switch can't control them as we expect.

Reply:
Yes, if we don't want PD to provide the service after tso server is stopped.